lib/src/main/java/uk/gov/di/ipv/core/library/service/CredentialIssuerService.java

```diff
@@ -40,6 +40,7 @@
 public class CredentialIssuerService {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(CredentialIssuerService.class);
+    private static final String API_KEY_HEADER = "x-api-key";
```

Comment:
Are we free to chose a header field or is this restricted to api gateway implementation? Just wondering because KBV we're asking which header field we intend to use so they can be consistent.

Reply:
This seemed to be the one that the AWS docs said to use. I'm not aware of another header that we can use